pkg/network/network_filter.go

```diff
@@ -156,13 +163,16 @@ func IsExcludedConnection(scf []*ConnectionFilter, dcf []*ConnectionFilter, conn
 		return false
 	}
 
+	buf := ipBufferPool.Get().([]byte)
```

Comment:
Why a `sync.Pool` here and not just a single slice, like the other files?

Reply:
It is just because this is not a method, and thus I don't have a way to store the buffer.
I think I could replace it by a global variable in case this function doesn't need to be thread-safe. WDYT?

Reply:
Realized that everything needs to be thread-safe since all methods are in the code path of `GetActiveConnections` which in turn is attach to a mux handler (and thus can process multiple HTTP connections concurrently)